charts/telepresence/values.yaml

```diff
@@ -52,6 +52,10 @@ affinity: {}
 ##
 priorityClassName: ""
 
+## The default registry to pull misc images
+tools:
+   registry: docker.io
```

Comment:
If you could also put the config [under `hooks`](https://github.com/telepresenceio/telepresence/blob/release/v2/charts/telepresence/values.yaml#L258) instead of tools (it's been newly created) that would be super please